examples/io/create_binaries.cpp

```diff
@@ -31,36 +38,36 @@ int create_binaries(const std::string& detector_file,
          event < common_opts.events + common_opts.skip; ++event) {
 
         // Read the cells from the relevant event file
-        traccc::cell_container_types::host cells_csv =
-            traccc::read_cells_from_event(event, common_opts.input_directory,
-                                          common_opts.input_data_format,
-                                          surface_transforms, digi_cfg,
-                                          host_mr);
+        traccc::cell_container_types::host cells_csv = traccc::io::read_cells(
+            event, common_opts.input_directory, common_opts.input_data_format,
+            &surface_transforms, &digi_cfg, &host_mr);
```

Comment:
Hmm, I'm not a huge fan of using pointers here instead of references, although I understand the use in allowing us to read cells without digitization configs for example. Out of curiosity, do we use this mechanism – passing nullpointers here – anywhere in the code?

Reply:
Have a look at `ConnectedComponentAnalysisTests`. :wink: That one does:

```c++
        traccc::cell_container_types::host data =
            traccc::io::read_cells(file_hits);
```

Reply:
Fair.